tests/resources/VCH-Util.robot

```diff
@@ -787,3 +792,19 @@ Manually Cleanup VCH
     ${out}=  Run Keyword And Ignore Error  Run  govc datastore.rm ${vch-name}
     ${out}=  Run Keyword And Ignore Error  Run  govc host.portgroup.remove ${vch-name}-bridge
     ${out}=  Run Keyword And Ignore Error  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run  govc object.destroy vm/${vch-name}
+
+Create Ops User Account On Test Server
+    Log To Console  \nChecking ops user account...
+    ${ops-username}=  Split String  %{VCH_OPS_USERNAME}  @
+    ${ops-username}=  Get From List  ${ops-username}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  sshpass -p vmware ssh -o StrictHostKeyChecking=no root@%{TEST_URL} /usr/lib/vmware-vmafd/bin/dir-cli user find-by-name --account ${ops-username} --login %{TEST_USERNAME} --password %{TEST_PASSWORD}
```

Comment:
`root` => `%{SSH_TEST_USERNAME}`
`vmware` => `%{SSH_TEST_PASSWORD}`

Make sure to update SSH secrets in `.drone.yml` too.